src/main/java/pl/betoncraft/betonquest/exceptions/HookException.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Created on 01.07.2018.
+ */
+package pl.betoncraft.betonquest.exceptions;
+
+import org.bukkit.plugin.Plugin;
+
+/**
+ * Thrown if BetonQuest tries to hook a plugin
+ */
+public class HookException extends Exception {
+
+    private static final long serialVersionUID = 7965553395053833302L;
+
+    /**
+     * The plugin
+     */
+    private final Plugin plugin;
+
+    /**
+     * Constructs a new exception
+     *
+     * @param plugin The plugin
+     */
+    public HookException(final Plugin plugin, final String message) {
```

Comment:
Maybe also add a constructor that takes an exception in case the hooking throws an exception?